examples/step-3/doc/results.dox

```diff
@@ -224,3 +224,209 @@ suggestions:
+insights on what is possible.
+<ul>
+<li> Changing the output from .gnuplot to .h5:</p>
+     To fully make use of the automation we first need to introduce a private variable for the number of 
+     global refinement steps <code> int n_refinement_steps </code>, which will be used for the output filename. 
+     In <code> make_grid() </code> we replace <code> triangulation.refine_global(5); </code> with
+     @code
+     n_refinement_steps = 5;
+     triangulation.refine_global(n_refinement_steps);
+     @endcode
+     So far, HDF5 only supports parallel output with MPI, for this reason we need to initialize an MPI
+     communicator with only one processor. This is done by adding the following code.
+     @code
+     int main(int argc, char* argv[])
+     {
+       Utilities::MPI::MPI_InitFinalize mpi_initialization(argc,argv,1);
+       ...
+     }
+     @endcode
+     Next we change the output routine as described in the namespace documentation.
+     @code
+     std::ofstream filename_h5;
+     filename_h5 << "solution_" << n_refinement_steps << ".h5";
+     DataOutBase::DataOutFilterFlags flags(true,true);
+     DataOutBase::DataOutFilter data_filter(flags);
+     data_out.write_filtered_data(data_filter);
+     data_out.write_hdf5_parallel(data_filter,filename_h5.str(),MPI_COMM_WORLD);
+     @endcode
+     
+     
+<li> Adding the point value and the mean (see extension above) into the .h5 file: </p>
+     After output of the solution, the file can be opened again to include more datasets.
+     This allows us to keep all the necessary information of our experiment in a single result file, 
+     which can then be read and processed by some postprocessing script.<br>
+     Have a further look at HDF5::Group::write_dataset for further information on the possible output options.<br>
+     First we include <code> "<deal.II/base/hdf5.h>" </code> into our file. Adding the following lines to the end 
+     of our output routine adds the additional information to our HDF5 file.
+     @code
+     HDF5::File data_file(filename_h5.str() , HDF5::File::FileAccessMode::open , MPI_COMM_WORLD );
+     Vector<double> point_value(1);
+     point_value[0] = VectorTools::point_value (dof_handler, solution,Point<2>(1./3, 1./3));
+     data_file.write_dataset("point_value",point_value);
+     Vector<double> mean_value(1);
+     mean_value[0] = VectorTools::compute_mean_value (dof_handler,QGauss<2>(fe.degree + 1),solution, 0);
+     data_file.write_dataset("mean_value",mean_value);
+     @endcode
+     
+<li> Using R and ggplot2 to generate PDFs with plots of the grid, the solution and convergence curves: </p>
+     If you are unfamiliar with R and ggplot2 you could check out the data carpentry course on R
+     <a href=https://datacarpentry.org/R-ecology-lesson/index.html>link</a>. <br>
+     Since "R + search term" does not yield good results for most search engines a specialized service 
+     <a href=rseek.org> RSeek </a> exists.
```

Comment:
This sentence is a bit awkward. I recommend replacing it with something like
```
Furthermore, since most search engines struggle with searches of the form "R + topic",
we recommend using the specialized service RSeek instead.
```
where RSeek is a full URL link (doxygen needs the full address).